src/_gettsim_tests/test_arbeitsl_geld.py

```diff
@@ -1,57 +1,38 @@
-import pandas as pd
 import pytest
 from _gettsim.interface import compute_taxes_and_transfers
 from pandas.testing import assert_series_equal
 
-from _gettsim_tests import TEST_DATA_DIR
 from _gettsim_tests._helpers import cached_set_up_policy_environment
+from _gettsim_tests._policy_test_utils import PolicyTestData, load_policy_test_data
 
-INPUT_COLS = [
-    "p_id",
-    "hh_id",
-    "tu_id",
-    "bruttolohn_vorj_m",
-    "wohnort_ost",
-    "kind",
-    "anwartschaftszeit",
-    "arbeitssuchend",
-    "sozialv_pflicht_5j",
-    "m_durchg_alg1_bezug",
-    "arbeitsstunden_w",
-    "alter",
-    "geburtsjahr",
-    "jahr",
-]
-YEARS = [2010, 2011, 2015, 2019]
+data = load_policy_test_data("arbeitsl_geld")
 
 
-@pytest.fixture(scope="module")
-def input_data():
-    file_name = "arbeitsl_geld.csv"
-    out = pd.read_csv(TEST_DATA_DIR / file_name)
-    return out
-
-
-@pytest.mark.parametrize("year", YEARS)
+@pytest.mark.parametrize(
+    ("test_data", "column"),
+    data.parametrize_args,
+    ids=str,
```

Comment:
I would love to see our own ids-function, so we can map back immediately to the test data.

Current output on failure is:

```
FAILED src/_gettsim_tests/test_arbeitsl_geld.py::test_arbeitsl_geld[2015-01-01 hh_id_5-arbeitsl_geld_m] - AssertionError: Series are different
FAILED src/_gettsim_tests/test_arbeitsl_geld.py::test_arbeitsl_geld[2019-01-01 hh_id_1-arbeitsl_geld_m] - test_arbeitsl_geld[2015-01-01 hh_id_5-arbeitsl_geld_m]
test_arbeitsl_geld[2019-01-01 hh_id_1-arbeitsl_geld_m]
test_arbeitsl_geld[2019-01-01 hh_id_2-arbeitsl_geld_m]
test_arbeitsl_geld[2010-01-01 hh_id_6-arbeitsl_geld_m]
```

What I would love to see would be:
```
test_arbeitsl_geld[2015.yaml, hh_id_5, arbeitsl_geld_m]
test_arbeitsl_geld[2019.yaml, hh_id_1, arbeitsl_geld_m]
test_arbeitsl_geld[2019.yaml, hh_id_2, arbeitsl_geld_m]
test_arbeitsl_geld[2010.yaml, hh_id_6, arbeitsl_geld_m]
```
i.e. each id might be `<file>, <testcase_id>, <column>`.

Probably just a matter of adding a method to `PolicyTestSet`, if I understand the `idfn`-example [here](https://docs.pytest.org/en/7.1.x/example/parametrize.html#different-options-for-test-ids) correctly.

Reply:
The callable specified in IDs gets applied to all arguments. This includes the column, which is a string. It can, therefore, unfortunately not be a method of `PolicyTestSet`.

Reply:
Should be close enough now:

```
test_arbeitsl_geld[2015.yaml hh_id_5-arbeitsl_geld_m]
test_arbeitsl_geld[2019.yaml hh_id_1-arbeitsl_geld_m]
test_arbeitsl_geld[2019.yaml hh_id_2-arbeitsl_geld_m]
test_arbeitsl_geld[2010.yaml hh_id_6-arbeitsl_geld_m] 
```